stl/inc/format

```diff
@@ -2925,23 +3060,23 @@ wstring vformat(const locale& _Loc, const wstring_view _Fmt, const wformat_args
 #undef _TEMPLATE_INT_0_NODISCARD // TRANSITION, VSO-1433873
 
 template <class... _Types>
-_NODISCARD string format(const string_view _Fmt, const _Types&... _Args) {
-    return _STD vformat(_Fmt, _STD make_format_args(_Args...));
+_NODISCARD string format(const _Fmt_string<_Types...> _Fmt, const _Types&... _Args) {
+    return _STD vformat(_Fmt._Str, _STD make_format_args(_Args...));

```

Comment:
P2216R3 depicts changes to `format_to_n()` and `formatted_size()` which are missing here.

Reply:
hmm, it occurs to me that there's no escape hatch for format_to_n or formatted size, this is annoying in our tests, but could be reasonable behavior as user-controlled format strings for these functions are a good way to get buffer overruns.

Reply:
I confirmed on the libraries mailing list that the lack of an escape hatch is intentional (I think victor views the vformat escape hatch as somewhat crude)